core/src/main/resources/error/error-classes.json

```diff
@@ -341,6 +346,11 @@
     ],
     "sqlState" : "22023"
   },
+  "STREAM_CLOSED" : {
+    "message" : [
+      "<streamType> stream is closed."
```

Comment:
This doesn't really like a user-facing error. I think IO errors are all internal errors? cc @srielau

Reply:
@srielau could you confirm please, based on the update I will update the changes

Reply:
I can't make that call. Why could the stream be closed? Would you expect a Jira ticket to be opened to investigate this error? What is the customer action? What is the Databricks action....